README.md

```diff
@@ -123,15 +123,40 @@ Notice that the linting commands now are nested into the `linters` object. The f
 
 #### Options
 
-* `concurrent` — _true_ — runs linters for each glob pattern simultaneously. If you don’t want this, you can set `concurrent: false`
-* `chunkSize` — Max allowed chunk size based on number of files for glob pattern. This option is only applicable on Windows based systems to avoid command length limitations. See [#147](https://github.com/okonet/lint-staged/issues/147)
 * `globOptions` — `{ matchBase: true, dot: true }` — [micromatch options](https://github.com/micromatch/micromatch#options) to
   customize how glob patterns match files.
 * `ignore` - `['**/docs/**/*.js']` - array of glob patterns to entirely ignore from any task.
 * `linters` — `Object` — keys (`String`) are glob patterns, values (`Array<String> | String`) are commands to execute.
-* `subTaskConcurrency` — `1` — Controls concurrency for processing chunks generated for each linter. This option is only applicable on Windows. Execution is **not** concurrent by default(see [#225](https://github.com/okonet/lint-staged/issues/225))
 * `relative` — `false` — if `true` it will give the relative path from your `package.json` directory to your linter arguments.
 
+## Using JS functions to customize linter commands
```

Comment:
Lets add the signature of the function to the docs, too.

Reply:
Also added an example of custom globs, as per https://github.com/okonet/lint-staged/issues/644